src/sections/maibornwolff.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ojemba</title>
+</head>
+<body>
+    <section class="pb-40">
+        <div class="mt-48 mb-20 ml-36 w-1/3">
+            <h1 class="font-railwayBold text-3xl">MaibornWolff x The Gym x Maliyo</h1>
+            <p class="font-railwayRegular text-lg py-4">
+                Für das Training werden die Erfahrung und das 
+                Wissen über erfolgreiche Softwareprojekte 
+                kombiniert mit dem standortspezifischen ...
+            </p>
+        </div>
+
+        <div class="ml-80 flex gap-40">
+            <div>
+                <div>
+                    <img class="w-80 h-44" src="/images/Logo MW.webp" alt="Logo of MaibornWolff">
+                </div>
+
+                <div class="mt-36 mb-44">
+                    <img class="w-80 h-32" src="/images/Maliyo Games.webp" alt="Maliyo Games logo">
+                </div>
+
+                <div class="grid justify-items-center">
+                    <img class="w-60 h-44" src="/images/The Gym logo.webp" alt="The Gym logo">
+                </div>
+            </div>
+            
+            <div class="flex flex-col gap-44 align-bottom font-railwayRegular text-base">
+                <div>
+                    <div class="flex">
+                        <img class="w-16 h-16" src="/images/image 1.webp" alt="BMW logo">
+                        
+                        <img class="w-44 h-11" src="/images/stihl.svg" alt="STIHL logo">
+                        
+                        <img class="w-32 h-16" src="/images/Miele_Logo_M_Red_sRGB.svg" alt="Miele logo">                  
+                    </div>
+                    <div class="w-2/5 mt-3">
+                        <p>
+                            haben Produkte und Prozesse vom   
+                            <span class="font-railwayBold">internationaler IT Dienstleister</span> digitalisieren 
+                            lassen
```

Comment:
- [ ]  kindly consider breaking this paragraph(adding `br` tags) as it is in the Figma design to give the expected design